.github/workflows/ubuntu_build.yml

```diff
@@ -71,10 +71,21 @@ jobs:
         run: |
           ./build.sh
 
+      - name: 'upload release build artifacts'
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+        with:
+          name: mtl-release-bin
+          path: '${{ github.workspace }}/build/*'
+
       - name: Build with debug
         run: |
           rm build -rf
           ./build.sh debug
+      - name: 'upload debug build artifacts'
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+        with:
+          name: mtl-debug-bin
+          path: '${{ github.workspace }}/build/*'
```

Comment:
Please remove this whole step from PR - we do not want to upload artifacts 2x times for each run with debugging symbols included, BDBA should get the release version of binaries inside zip archive as far as I remember.

Reply:
done